### PR TITLE
WDP200330-33

### PR DESCRIPTION
**Pull request by jurecki**
New Component ClientFeedback has been added. 
Based on design https://uploads.kodilla.com/bootcamp/wdp/group-project-01/section-feedback.png

### CODE REVIEW

File: src/components/features/NewFurniture/NewFurniture.js [hunk 1]
```diff
@@ -122,6 +121,34 @@ class NewFurniture extends React.Component {
...
+                  <li>
+                    <a> </a>
+                  </li>
+                  <li>
+                    <a> </a>
+                  </li>
+                  <li>
+                    <a> </a>
+                  </li>
+                </ul>
```
Comment (ichtys77) [inline src/components/features/NewFurniture/NewFurniture.js]: <ul>
{dots}
</ul>

and the first dot should be orange

File: src/components/layout/ClientFeedback/ClientFeedback.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+const ClientFeedback = ({ id, name, status, picture, description }) => (
+  <div className={styles.root}>
+    <FontAwesomeIcon className={styles.icon} icon={faQuoteRight} />
+    <div className={styles.description}>{description}</div>
+    <div className={styles.user}>
+      <img className={styles.user_photo} src={picture} alt={name} />
+      <div className={styles.user_description}>
+        <div>
+          {' '}
+          <strong>{name}</strong>
```
Comment (ichtys77) [inline src/components/layout/ClientFeedback/ClientFeedback.js]: You can use class style

Comment (jurecki) [reply src/components/layout/ClientFeedback/ClientFeedback.js]: It's mistake, I'll remove it

File: src/components/features/NewFurniture/NewFurniture.js (hunk 1 above)
Comment (jurecki) [reply src/components/features/NewFurniture/NewFurniture.js]: ok... but in this case, I have to make only design without function of changing 'feedback'. But why not try :) 

I've push new commit with 'management' of display feedback. 
{dots} change products, that I have to make "new" dots.

Comment (ichtys77) [reply src/components/features/NewFurniture/NewFurniture.js]: below it's a quote from task description in Jira ;):

> Tak jak w innych elementach tego typu, kropka przy aktywnym slajdzie i na hover ma być pomarańczowa.

File: src/components/features/NewFurniture/NewFurniture.js [hunk 3]
```diff
@@ -75,6 +80,20 @@ class NewFurniture extends React.Component {
...
 
+    const dotsFeedback = [];
+    for (let i = 0; i < feedbackCount; i++) {
+      dotsFeedback.push(
+        <li>
+          <a
+            onClick={() => this.handlePageChangeFeedback(i)}
+            className={i === activePageFeedback && styles.active}
+          >
+            page {i}
```
Comment (Varenthein) [inline src/components/features/NewFurniture/NewFurniture.js]: Why this `<` sign is in the new line?

Comment (jurecki) [reply src/components/features/NewFurniture/NewFurniture.js]: I copied it from: 
`const dots = [];
    for (let i = 0; i < pagesCount; i++) {
      dots.push(
        <li>
          <a
            onClick={() => this.handlePageChange(i)}
            className={i === activePage && styles.active}
          >
            page {i}
          </a>
        </li>
      );
    }`

change everywhere?

Comment (jurecki) [reply src/components/features/NewFurniture/NewFurniture.js]: fixed

Comment (jurecki) [reply src/components/features/NewFurniture/NewFurniture.js]: I've tried fixed it, but probably 'prettier code" plugin has  changed ">" to new line.

**Pull request merged**